locals.tf

```diff
@@ -1,3 +1,46 @@
 locals {
-  
+    path = "${path.module}/queries"
+
+    law_name = reverse(split("/", var.law_id))[0]
+    law_rg   = split("/", var.law_id)[4]
+
+    rules = {
+        "alr-prd-heartbeat-ux-law-perf-crit-01": {
+            description = "Alert when Heartbeat Stopped"
+            query_path = "${local.path}/unix_heartbeat.kusto"
+            time_window = 2280
+        }
+        "alr-prd-diskspace-ux-law-perf-warn-crit-01": {
+            description = "Alert when Filespace runs out of space"
+            query_path = "${local.path}/unix_filespace.kusto"
+            time_window = 2280
+        }
+        "alr-prd-diskspace-ux-law-perf-warn-crit-02": {
+            description = "Alert when SAP Filespace runs out of space"
+            query_path = "${local.path}/unix_sap_filespace.kusto"
+            time_window = 2280
+        }
+
+        "alr-prd-eventlog6008-win-law-event-crit-01": {
+            description = "Alert when the Windows event log entries occurred"
+            query_path = "${local.path}/windows_event_6008.kusto"
+            time_window = 30
+        }
+        "alr-prd-eventlog55-win-law-event-crit-01": {
+            description = "Alert when the Windows event log entries occurred"
+            query_path = "${local.path}/windows_event_55.kusto"
+            time_window = 30
+        }
```

Comment:
Please include meaningful descriptions, so it is known, what is monitored and why. At least include the eventlog numbers.

Reply:
Copied description from monitoring team. Will change